src/libs/actions/Report.js

```diff
@@ -191,6 +191,7 @@ function getSimplifiedIOUReport(reportData) {
  * @param {Object} report
  * @param {Object[]} report.reportActionList
  * @param {Number} report.reportID
+ * @return {Promise}
```

Comment:
Ah, aren't we currently avoiding returning Promises from Actions?

Reply:
Not if they are being called by other actions IIRC.

Reply:
>Public action methods should never return anything

So, as long as this method remains private then it doesn't break the philosophy

Reply:
But I believe the main pattern we are trying to avoid with that is having the UI wait for a promise to resolve and then handle the response to update some local state.

Reply:
Makes sense 👍